src/App/Multisite/Site.php

```diff
@@ -0,0 +1,195 @@
+<?php
+
+/**
+ * Site Entity
+ *
+ * @author     Ushahidi Team <team@ushahidi.com>
+ * @package    Ushahidi\Platform
+ * @copyright  2014 Ushahidi
+ * @license    https://www.gnu.org/licenses/agpl-3.0.html GNU Affero General Public License Version 3 (AGPL3)
+ */
+
+namespace Ushahidi\App\Multisite;
+
+use Illuminate\Support\Facades\Cache;
+use Illuminate\Support\Facades\DB;
+use Ushahidi\Core\Entity\ConfigRepository;
+
+// @todo consider just an Eloquent model? or ushahidi entity
```

Comment:
Good question -- I'm not sure how we'd handle the isDBReady stuff if this were an Eloquent model. Split it out?